app/src/main/java/com/cp/tasky/agenda/overview/domain/OverviewUseCase.kt

```diff
@@ -0,0 +1,68 @@
+package com.cp.tasky.agenda.overview.domain
+
+import com.cp.tasky.agenda.overview.domain.model.Overview
+import com.cp.tasky.agenda.shared.data.model.Event
+import com.cp.tasky.agenda.shared.data.model.Items
+import com.cp.tasky.agenda.shared.data.model.Reminder
+import com.cp.tasky.agenda.shared.data.model.Task
+
+object AgendaOverviewUseCase {
+
+    private const val DEFAULT_INITIALS = "NA"
+
+    /**
+     * All agenda items should be ordered
+     * ascending by time
+     */
+    fun sortAgendaItemsByAscendingTime(items: Overview): List<Items> {
+        val tempList = mutableListOf<Items>()
```

Comment:
What is `Items`? That doesn't seems so clear to me. Also if you're having a list of items, it sounds like having a list of lists (since items is already plural and you have a list of those sets of items)